testing/Makefile

```diff
@@ -37,21 +64,36 @@ else
 endif
 	@echo "==> Starting containers"
 	mkdir -p $(MINIO_DATA_DIR)
-	@. ./.env && CURRENT_UID=$(CURRENT_UID) CURRENT_GID=$(CURRENT_GID) $(COMPOSE) up -d
+	@. ./.env && CURRENT_UID=$(CURRENT_UID) CURRENT_GID=$(CURRENT_GID) $(COMPOSE) -f docker-compose.yaml -f $(COMPOSE_FILE) up -d dc1_node_1 dc2_node_1
 	$(COMPOSE) exec -T --privileged dc1_node_1 sudo bash -c 'echo "fs.aio-max-nr = 1048579" > /etc/sysctl.d/50-scylla.conf'
 	$(COMPOSE) exec -T --privileged dc1_node_1 sudo sysctl -p /etc/sysctl.d/50-scylla.conf
-	@echo "==> Waiting for cluster"
-	@until [ 6 -eq $$($(NODETOOL) status | grep -c "UN") ]; do echo "..."; sleep 2; done
-	@./nodes_exec rm /root/.cqlshrc
+	@echo "==> Waiting for first nodes from DC1 and DC2 to spin up"
+	@until [ 2 -le $$($(NODETOOL) status | grep -c "UN") ]; do echo -n "."; sleep 2; done ; echo ""
+	@echo "==> First nodes from DC1 and DC2 are ready"
+	@. ./.env && CURRENT_UID=$(CURRENT_UID) CURRENT_GID=$(CURRENT_GID) $(COMPOSE) -f docker-compose.yaml -f $(COMPOSE_FILE) up -d dc1_node_2 dc2_node_2
+	@echo "==> Waiting for second nodes from DC1 and DC2 to spin up"
+	@until [ 4 -le $$($(NODETOOL) status | grep -c "UN") ]; do echo -n "."; sleep 2; done ; echo ""
+	@echo "==> Second nodes from DC1 and DC2 are ready"
+	@. ./.env && CURRENT_UID=$(CURRENT_UID) CURRENT_GID=$(CURRENT_GID) $(COMPOSE) -f docker-compose.yaml -f $(COMPOSE_FILE) up -d
+	@echo "==> Waiting for sm db cluster"
+	@until [ 1 -le $$($(SM_NODETOOL) status | grep -c "UN") ]; do echo -n "."; sleep 2; done ; echo ""
+	@echo "==> SM cluster is ready"
+
+	@echo "==> Waiting for rest of the cluster"
+	@until [ 6 -le $$($(NODETOOL) status | grep -c "UN") ]; do echo -n "."; sleep 2; done ; echo ""
+	@echo "==> Cluster is ready"
```

Comment:
Why to wait for dc1_node_1 and dc2_node_1 first ? For *node_2 second and the rest at the end ?
What is the idea behind it ?

Reply:
Can we bring back the previous approach ?

Reply:
Scylla bootstrapping is very fragile, in general, it is not recommended to bootstrap more than one node at a time.
Since it is brand new, empty cluster we get away with bootstraping non-seed nodes in parallel.
But not with seeds nodes, since timeout on seeds is way stricter.
Locally I have 60% chance of failing bootstrap with the way we did it before.